.github/workflows/build.yml

```diff
@@ -59,12 +59,14 @@ jobs:
       - name: Get Latest RT Release
         id: get-latest-rt
         run: |
+          set -o pipefail
```

Comment:
I think we should not do that. It is intended sometime to have failed pipe in our script.
Instead you should add a check on the variable you need to not be empty

Reply:
In the scope of this step I think it's safe to add this, since this step is responsible for getting the release tag only. This doesn't affect other steps.

Reply:
I guess I can live with that, but I feel it is not ideal:
I'm pretty sure in 6 months when someone will need to change the CI, they will see this line `set -o pipefail` not knowing what it does... and more importantly why it is there and will get stuck with it not knowing what to do

if you want something short, you can do
```
[[ ! -z "$LATEST_RUNTIME_RELEASE" ]]
```
This will ensure your variable is set and can easily understand WHY there is this line, even if you have to look for what [[ and -z mean

Reply:
At best, leave the `set -o pipefail` but add a comment like:
# This step expects all variable to be set by ensuring all piped command succeeds

Reply:
We can work with `[[ ! -z "$LATEST_RUNTIME_RELEASE" ]]` that's fine too, the idea is just that if a step has `needs: get-sha` defined it's not a false positive when it doesn't have the variable set.

I was under the impression that `set -eo pipefail` was a pretty well understood semantic for bash scripts.